debian/oxen-node.openrc

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh
+# shellcheck shell=sh # Written to comply with IEEE Std 1003.1-2017
+# shellcheck disable=SC2034 # Krey: OpenRC is using variables that are not expected to be set
+# Generated by sysd2v v0.3  --  http://www.trek.eu.org/devel/sysd2v
+# kFreeBSD do not accept scripts as interpreters, using #!/bin/sh and sourcing.
+if [ true != "$INIT_D_SCRIPT_SOURCED" ] ; then
+    set "$0" "$@"; INIT_D_SCRIPT_SOURCED=true . /lib/init/init-d-script
+fi
+### BEGIN INIT INFO
+# Provides:       oxen-node
+# Required-Start: $remote_fs
+# Required-Stop:  $remote_fs
+# Should-Start:   $network
+# Should-Stop:    $network
+# X-Start-Before: loki-storage
+# X-Stop-After:   loki-storage
+# Default-Start:  2 3 4 5
+# Default-Stop:   0 1 6
+# Description:    Oxen network node
+### END INIT INFO
+set -a
+TERM=xterm
+set +a
+
+DESC="oxen-node"
+DAEMON="/usr/bin/oxend"
+DAEMON_ARGS="--non-interactive --config-file /etc/oxen/oxen.conf"
+PIDFILE="/var/run/oxen-node-sysd2v.pid"
```

Comment:
Any particular reason for this pid filename convention?

Reply:
Not really.. This part was generated by the mentioned script

Reply:
Where afaik the concern are systems with both systemd compatibility and openrc runner?

Reply:
No, systemd doesn't write a pidfile, and AFAIK systemd will ignore a sysvinit script in /etc/init.d/whatever as long as a systemd service file is installed.

Reply:
Seems sane for me to not use the `sysd2v` then.. Note script generated using `sysd2v` converter.